src/main/java/com/mercadopago/resources/payment/Payment.java

```diff
@@ -185,4 +186,10 @@ public class Payment extends MPResource {
 
   /** Point of interaction. */
   private PaymentPointOfInteraction pointOfInteraction;
+
+  /** 3DS. */
+  private static final String PaymentThreeDSecureMode = "not_supported";
```

Comment:
Aqui devemos apenas receber a informação da API e montar no nosso DTO, então vai deve ser static final, e não deve ter valor atribuído por padrão.
```suggestion
  private String PaymentThreeDSecureMode;
```